minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java

```diff
@@ -139,6 +139,7 @@ public void stop() throws Exception {
    * @throws Exception if master thread cannot be interrupted
    */
   public void kill() throws Exception {
+    mAlluxioMaster.stop();
```

Comment:
If we're going to change the semantics from kill to stop, we should update the name and javadoc and make sure the change makes sense at all the call sites.